patches/chrome-tools-build-win-create_installer_archive.py.patch

```diff
@@ -60,26 +60,37 @@ index b16b16bdc439014bf647f80ca9b7ce8215cfbb6c..cf36aac8b0d18fb0a1dd6f9018dcd53e
 +        rel_file = os.path.join(rel_dir, name)
 +        candidate = os.path.join('.', 'resources', 'brave_rewards' '_locales', rel_file)
 +        g_archive_inputs.append(candidate)
++
++def CopyPreSignedBinaries(output_dir, staging_dir, current_version):
```

Comment:
Do not patch in methods like this. Please include a py file from brave and just call the methods it defines. We have to reduce these patches

Reply:
Ok, will create separate file for this and above other methods also.